src/globus_sdk/_testing/models.py

```diff
@@ -13,6 +13,7 @@ class RegisteredResponse:
         "search": "https://search.api.globus.org/",
         "gcs": "https://abc.xyz.data.globus.org/api",
         "groups": "https://groups.api.globus.org/v2/",
+        "timer": "https://timer.automate.globus.org/",
```

Comment:
I was thinking about that. Maybe a stub `ActionsClient` would be reasonable. However, for the callback URL issue, it would still be a little odd, I think? Since there are several different actions, which have differing URLs. I think we'd need something like an Action Provider enum which stores a bunch of the URLs for various action providers. What do you think?